lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
> Just that it writes to stdout instead of undocumented.txt? That it affects jazzy’s exit code?

I was intending just to address these two.

> That this might do checks for things other than undocumented items?

Hadn't considered this. I suppose `--lint` might not be the correct name here then, because jazzy itself isn't doing any "linting" per se.